bpf/bpf_lxc.c

```diff
@@ -508,7 +508,7 @@ static __always_inline int handle_ipv4_from_lxc(struct __ctx_buff *ctx,
 						__u32 *dstID)
 {
 	struct ipv4_ct_tuple tuple = {};
-#ifdef ENABLE_ROUTING
+#ifndef ENABLE_ENDPOINT_ROUTES
```

Comment:
Can we please remove this one and just tell the compiler that the variable may be unused? Then it should automatically be compiled out if the variable is not needed. Like this:
```c
	union macaddr __maybe_unused router_mac = NODE_MAC;
```

Reply:
@qmonnet thank you for suggestion and it was addressed now.